recipes/kissfft/all/conanfile.py

```diff
@@ -0,0 +1,71 @@
+import os
+from conans import ConanFile, CMake, tools
+
+
+class KissfftConan(ConanFile):
+    name = "kissfft"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/mborgerding/kissfft"
+    description = "a Fast Fourier Transform (FFT) library that tries to Keep it Simple, Stupid"
+    topics = ("conan", "fft", "kiss", "frequency-domain", "fast-fourier-transform")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False],
+               "fPIC": [True, False],
+               "datatype": ["float", "double", "int16_t", "int32_t", "simd"],
+               "use_alloca": [True, False]}
+    default_options = {"shared": False,
+                       "fPIC": True,
+                       "datatype": "float",
+                       "use_alloca": False}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake",
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        cmake = CMake(self)
+        cmake.definitions["KISSFFT_PKGCONFIG"] = False
```

Comment:
`KISSFFT_OPENMP` could be set to False explicitly with a TODO to not forget to eventually offer this option in a new revision.